### PR TITLE
[TFF-220] Equipment Saving

### PR DESCRIPTION
**Pull request by thofmann**
### Summary

We can now save equipment to be used later. This PR does not include the ability to retrieve/unsave equipment.

### Checklist

- [x] Have you added unit and functional tests as appropriate?
- [x] Did you update/add documentation for new methods or changed functionality?
- [x] Have you merged the target branch down into this one?

### How to Review

1. Run both the server and client on `item-save-TFF-220`.
2. Create a model that can be checked out.
3. Check out a model and an item.
4. Save them both.

### Links

- [TFF-220](https://msoese.atlassian.net/browse/TFF-220)
- TheFourFifths/consus-client/pull/111

### CODE REVIEW

File: docs/Actions.md [hunk 1]
```diff
@@ -200,7 +202,7 @@ Checks out equipment and sets the equipments due date to the one provided.
 
 ### Data
 
-- `studentId`: 123456
+- `studentId`: The student checking out
```
Comment (mattmahn) [inline docs/Actions.md]: :+1:

File: docs/REST.md [hunk 2]
```diff
@@ -132,6 +134,8 @@ Adds a fault to a specified item.
 
 Get a list of all currently overdue items.
 
+### Sample Response
```
Comment (mattmahn) [inline docs/REST.md]: :100:

File: test/functional/save-item.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import ItemStore from '../../.dist/store/item-store';
+import ModelStore from '../../.dist/store/model-store';
+
+describe('Save Item', () => {
+
+    before(() => {
+        return server.start(8080);
+    });
+
+    after(() => {
```
Comment (mattmahn) [inline test/functional/save-item.js]: weirdo not putting it in execution-order

Comment (thofmann) [inline test/functional/save-item.js]: I understand the concern, but I think there's a fair case for group `before` with `after` and `beforeEach` with `afterEach`.

This example is showing them in this order (`before` -> `after` -> `beforeEach`): https://mochajs.org/#hooks

**Pull request merged**